companion/src/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-PROJECT( companion )
```

Comment:
Hmm, I was wrong above the PROJECT_NAME is used also in the name of DEB package and it must have different value from the Companion 2.0. We used `companion` for 2.0, I would like to use `companion2.1` for 2.1. The package name must be different if we want to have both versions installed at the same time!